ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/BuyerPhone.kt

```diff
@@ -1,4 +1,4 @@
-package com.braintreepayments.api
+package com.braintreepayments.api.shopperinsights
```

Comment:
Package on all of these should be `com.braintreepayments.api` so that we can leverage package-private classes/methods across the entire SDK (everything lives in `com.braintreepayments.api`)

Reply:
To clarify - the directory setup should be 
`braintree_android/ShopperInsights/src/main/java/com/braintreepayments/api/BuyerPhone.kt`

Reply:
> Package on all of these should be `com.braintreepayments.api` so that we can leverage package-private classes/methods across the entire SDK (everything lives in `com.braintreepayments.api`)

Good to know.
Package-private works for java code, but how does that look for Kotlin code?
`internal` at a module level, so I'm not sure how the code should look.

Reply:
It might not actually apply to any classes in this module currently, since they look like they can all be public or internal currently, but for Kotlin classes that we want to use across modules, we use the `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)` annotation (ex: BraintreeClient).